docs/checkouts/overview.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Checkouts Overview
+sidebar_label: Overview
+description: What are Checkouts
+keywords:
+  - checkouts
+  - features
+  - lifecycle
+---
+
+# Checkouts: Unified Fintech Checkout™
+
+## Overview
+
+The JustiFi Unified Fintech Checkout (UFC) is a streamlined API solution designed to integrate
+embedded payments, insurance, and "Buy Now, Pay Later" (BNPL) options within a single checkout.
```

Comment:
I'd switch '"Buy Now, Pay Later" (BNPL)' to 'BNPL (Buy Now Pay Later)' because the acronym seems very well known at this point.